sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1842,14 +2067,17 @@ class ReadFromBigQuery(PTransform):
     default.
 
   Args:
+    method: The method to use to read from BigQuery. It may be
```

Comment:
Could we specify what's limitation of `DIRECT_READ`? I believe in case of `DIRECT_READ` query parameter is not used

Reply:
Added some more details in the documentation.